src/main/webapp/js/user-page-loader.js

```diff
@@ -139,6 +143,42 @@ function addCancleButtonFunction(editButton, cancleButton, saveButton, messageDi
   messageDiv.appendChild(cancleButton);
 }
 
+/*adding functionality to save button and append it to message div*/
+function addSaveButtonFunction(editButton, cancleButton, saveButton, messageDiv, bodyDiv, bodyText, message) {
+  saveButton.innerHTML = "Save";
+  saveButton.onclick = function() {
+    editButton.style.visibility="visible";
```

Comment:
Move changing editButton style to within the try block on line 170, so that changes to the buttons happen in the same place.